src/pages/Encounters/ReportPreview.tsx

```
@@ -0,0 +1,483 @@
+              </div>
+            ) : (
+              <div className="flex flex-col">
+                {reports.map((report) => (
+                  <button
+                    key={report.id}
+                    onClick={() => setSelectedReportId(report.id)}
+                    className={cn(
+                      "flex flex-col gap-1 px-3 py-2.5 text-left border-b border-gray-100 transition-colors hover:bg-gray-50",
+                      selectedReportId === report.id && "bg-primary-50",
+                    )}
+                  >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)